activerecord/lib/active_record/migration.rb

```diff
@@ -1170,6 +1174,14 @@ def load_migrated
 
     private
 
+    def database_connection
```

Comment:
No -- I'm going to add `:nodoc:` here. The only new public API for users is `ActiveRecord::Migration#connection`.